### PR TITLE
Custom Yup locale translations for IT, DE and EN

### PR DESCRIPTION
**Pull request by mmalfertheiner**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          tag_name: ${{ github.ref }}
+          release_name: Release ${{ github.ref }}
+          draft: false
+          prerelease: false
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '12.x'
```
Comment (alexlanz) [inline .github/workflows/main.yml]: Is version 12 still up to date?

Comment (alexlanz) [reply .github/workflows/main.yml]: Didn't we upgrade to version 16

File: readme.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+npm install @aboutbits/yup-locales
+```
+
+Second, you add the custom language packages to your project.
+
+```js
+import { setLocale } from 'yup'
+import { en, it, de } from '@aboutbits/yup-locales'
+import { LocaleObject } from 'yup/lib/locale'
+import { Language } from '../components/support/types'
```
Comment (alexlanz) [inline readme.md]: Are these two required here in the example?

**Review by alexlanz — APPROVED** [review]
Looks good! I did not check every translation, because I think we will check and optimize these with the time.

File: .github/workflows/main.yml (hunk 1 above)
Comment (mmalfertheiner) [reply .github/workflows/main.yml]: Yes, I copied this from the internationalization package. I will update there as well.

File: readme.md (hunk 2 above)
Comment (mmalfertheiner) [reply readme.md]: nope, thanks :)

**Pull request merged**